### PR TITLE
ZEPPELIN-506 Expand "create notebook" of REST API to have initial paragraphs

### PR DESCRIPTION
**Pull request by HeartSaVioR**
### What is this PR for?

To provide initial paragraphs when creating new notebook via REST API

### What type of PR is it?

Improvement

### Todos
* [ ] - 

### Is there a relevant Jira issue?

https://issues.apache.org/jira/browse/ZEPPELIN-506

### How should this be tested?

1. Request ```http://<zeppelin host>:<zeppelin port>/api/notebook``` with POST method including JSON payload, 

```
{
    "name": "Test for initial paragraphs2",
    "paragraphs": [
        {
            "text": "%spark\nvar hello: Int=1"
        },
        {
            "text": "%spark\nhello = hello + 1"
        },
        {
            "text": "%spark\nimport java.util.Date\n\nprintln(hello + \", current time is <\" + new Date + \">\")"
        }
    ]
}
```

2. Check notebook is made with initial paragraphs via UI or REST API or Web socket.

### Questions:

* Does the licenses files need update? (No)
* Is there breaking changes for older versions? (No, added parameter is optional.)
* Does this needs documentation? (Yes, I've updated related doc.)

### CODE REVIEW

File: docs/rest-api/rest-notebook.md [hunk 1]
```diff
@@ -93,7 +93,21 @@ limitations under the License.
     </tr>
     <tr>
       <td> sample JSON input </td>
-      <td><pre>{"name": "name of new notebook"}</pre></td>
+      <td><pre>
+{
+  "name": "name of new notebook", 
+  "paragraphs": [
```
Comment (Leemoonsoo) [inline docs/rest-api/rest-notebook.md]: Will there be any good way to mention 'paragraphs' is optional?

Comment (HeartSaVioR) [inline docs/rest-api/rest-notebook.md]: Maybe we want to describe parameters of json independently.
I'm also curious what's the best way to describe API's parameters in docs. Inner table would be fine? Do you have some other ideas?

Comment (Leemoonsoo) [inline docs/rest-api/rest-notebook.md]: I have no good idea for now. Maybe listing sample one with 'paragraphs'  another without 'paragraphs'?

Comment (HeartSaVioR) [inline docs/rest-api/rest-notebook.md]: Fine. I think adding description of parameters requires changing of all REST APIs, and we can file a new issue to JIRA.
I'll just add a new sample which doesn't have paragraphs.
Thanks for quick review!

**Comment by HeartSaVioR** [comment]
@Leemoonsoo Addressed your comment.

**Comment by Leemoonsoo** [comment]
@HeartSaVioR Thank you for very useful improvement.

Looks good to me

**Comment by prabhjyotsingh** [comment]
Tested looks good to me :+1:

**Comment by Leemoonsoo** [comment]
Merge if there're no more discussions

**Comment by HeartSaVioR** [comment]
@Leemoonsoo Thanks for merging!

**Pull request closed without merging**